src/test/java/com/godel/project/javamasteryproject/services/EmployeeServiceTest.java

```diff
@@ -0,0 +1,82 @@
+package com.godel.project.javamasteryproject.services;
+
+import com.godel.project.javamasteryproject.controllers.dto.EmployeeDto;
+import com.godel.project.javamasteryproject.services.interfaces.IEmployeeService;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.List;
+
+@SpringBootTest
+public class EmployeeServiceTest {
+
+    @Autowired
+    private IEmployeeService employeeService;
+
+    public EmployeeDto createEmployeeDto() {
```

Comment:
You can create local field EmployeeDto and set value in @Before method. It will set automatical before each test